AdminUi/src/AdminUi/ClientApp/src/app/components/client/client-edit/client-edit.component.ts

```diff
@@ -67,6 +67,7 @@ export class ClientEditComponent {
                     this.client = data.result;
                 }
                 this.displayClientSecretWarning = true;
+                this.disabled = true;
```

Comment:
No, because after create a client, there's no edit feature, hence the page's components become disabled so you can't actually attempt to save again.
The only thing left enabled is the cancel button to navigate outside the page.

Reply:
Makes sense